BlueChip/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "BlueChip",
+    "assets": [
+      {
+        "description": "The native staking token of BlueChip.",
+        "denom_units": [
+          {
+            "denom": "ubluechip",
+            "exponent": 0
+          },
+          {
+            "denom": "ebl",
+            "exponent": 6
+          }
+        ],
+        "base": "ubluechip",
+        "name": "blue chip",
+        "display": "bcp",
+        "symbol": "BCP",
+        "logo_URIs": {
```

Comment:
Please remove logo_URIs, as this is a legacy property we are no longer adding to